protocol/src/main/kotlin/tbdex/sdk/protocol/models/ResourceData.kt

```diff
@@ -27,7 +27,7 @@ class OfferingData(
   val payinCurrency: CurrencyDetails,
   val payinMethods: List<PaymentMethod>,
   val payoutMethods: List<PaymentMethod>,
-  val requiredClaims: PresentationDefinitionV2
+  val requiredClaims: PresentationDefinitionV2?
```

Comment:
hmm, the [spec](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#offering) says this isn't nullable? are we changing that currently

Reply:
we should change it, I think we've discussed that it should not be required. cc @mistermoe in case you can remember where we had that conversation.